shared/login/signup/common.js

```diff
@@ -29,7 +31,13 @@ export const Wrapper = (props: Props) => (
     >
       {props.children}
     </Box2>
-    <HeaderHocHeader onBack={props.onBack} headerStyle={styles.header} />
+    {flags.useNewRouter ? (
```

Comment:
`useNewRouter` wraps `HeaderHocHeader` with a `SafeAreaViewTop` which breaks `headerStyle`. Add with an extra box here to fix.